src/models/Restaurant.ts

```diff
@@ -0,0 +1,19 @@
+import {Schema, model} from 'mongoose';
+
+const RestaurantSchema = new Schema({
+    name: String,
+    image: String,
+    popularity: Number,
+    address: String,
+    from: String,
+    to: String,
+    openingDate: String, 
+    averagePrice: Number,
+    distance: Number,
+    chef: { type: Schema.Types.ObjectId, ref: 'Chef' },
```

Comment:
you chose to use two way referencing, you can also use one way referencing, both have pros and cons:
https://www.mongodb.com/docs/v7.0/tutorial/model-referenced-one-to-many-relationships-between-documents/#pattern